unit-tests/src/test/scala/java/util/ArraysTest.scala

```diff
@@ -1,18 +1,1173 @@
-package java.util
+// Ported from Scala.js commit: ba618ed dated: 2020-10-05
+
+package org.scalanative.testsuite.javalib.util
+
+import language.implicitConversions
 
-import org.junit.Test
 import org.junit.Assert._
+import org.junit.Assume._
+import org.junit.Ignore
+import org.junit.Test
+
+import scala.scalanative.junit.utils.AssertThrows._
+import org.scalanative.testsuite.utils.Platform._
+
+import java.util.{Arrays, Comparator}
+
+import scala.reflect.ClassTag
+
+object ArraysTest extends ArraysTest
 
 class ArraysTest {
+  // To invoke org.junit.Assert.assertArrayEquals on Array[T]
+  implicit def array2erasedArray[T](arr: Array[T]): Array[AnyRef] =
+    arr.map(_.asInstanceOf[AnyRef])
+
+  def Array[T: ClassTag](v: T*): scala.Array[T] = scala.Array(v: _*)
```

Comment:
Here I actually meant to remove the `def Array`, not just the comment. Then all the `Array(...)`s in the test will directly go to `scala.Array(...)` instead of this indirection (which was necessary in Scala.js but not in Native).

Reply:
Done. Review by a knowledgeable person does make the code better. Thank you.